examples/wait_for.php

```diff
@@ -99,13 +99,14 @@ function custom_wait_for_task(string $thing = 'foobar'): void
 function wait_for_docker_container_task(): void
 {
     try {
+        $checkLogSince = (new \DateTimeImmutable())->format(\DateTimeInterface::RFC3339);
```

Comment:
You can use the date fonction here

Reply:
You are right. Just fixed in the last commit. :-)